exporter/util.go

```diff
@@ -218,17 +218,24 @@ func (ic *importContext) IsUserOrServicePrincipalDirectory(path, prefix string,
 }
 
 func (ic *importContext) emitRepoByPath(path string) {
-	ic.Emit(&resource{
-		Resource:  "databricks_repo",
-		Attribute: "path",
-		Value:     strings.Join(strings.SplitN(path, "/", 5)[:4], "/"),
-	})
+	parts := strings.SplitN(path, "/", 5)
+	if len(parts) >= 4 {
+		ic.Emit(&resource{
+			Resource:  "databricks_repo",
+			Attribute: "path",
+			Value:     strings.Join(parts[:4], "/"),
+		})
+	} else {
+		log.Printf("[WARN] Incorrect Repos path")
```

Comment:
instead of warn, shouldn't we Fail here?

Reply:
Unfortunately, our APIs aren't strictly consistent, so it's possible to create jobs pointing to non-existing paths, clusters, etc., but we should tolerate that :-( Otherwise, a lot of exports will fail - it's easier to fix TF code sometimes than fix objects in the workspace